File: Data Structures & Algorithms/duplicate-integer/submission-1.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        HashSet<int> s = new HashSet<int> ();

        foreach (int num in nums){
            if(s.Contains(num)){
                return true;
            }
            s.Add(num);
        }
        return false;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
import heapq as hq
from collections import defaultdict

class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        
        q = []
        d = defaultdict(int)

        for num in nums:
            d[num]+=1

        for key in d:
            hq.heappush(q, [d[key], key])

            if len(q) > k:
                hq.heappop(q)


        result = []
        while q:
            result.append(hq.heappop(q)[1])

        return result
        
            
        
